cloud_sql/sqlserver_instance_pitr/main.tf

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+# [START cloud_sql_sqlserver_instance_pitr]
```

Comment:
While this is the correct region tag this sample should probably go with the other PITR samples (appended to https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/cloud_sql/instance_pitr/main.tf). That folder also has testing disabled, as this resource may not be able to be created as part of continuous CI. (Will verify by trying to validate this sample)

Reply:
Actually this is fine.